dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/RenormalizationProcessorTest.java

```diff
@@ -87,31 +107,133 @@ private int computeEnd(int position, String reference, String alternate) {
-        assertMatchesExpected(3, "", "G", 2, 2, "", "G");
-        assertMatchesExpected(3, "", "CG", 2, 3, "", "GC");
-        assertMatchesExpected(5, "", "CG", 4, 5, "", "GC");
-        assertMatchesExpected(7, "", "C", 6, 6, "", "C");
-        assertMatchesExpected(7, "", "CC", 6, 7, "", "CC");
-        assertMatchesExpected(7, "", "CCC", 6, 8, "", "CCC");
+        assertMatchesExpected(3, "", "G", 2, 2, "", "G");   // 2:G>GG
+        assertMatchesExpected(3, "", "CG", 2, 3, "", "GC"); // 2:G>GCG
+        assertMatchesExpected(5, "", "CG", 4, 5, "", "GC"); // 4:G>GCG
+        assertMatchesExpected(7, "", "C", 6, 6, "", "C");   // 6:C>CC
+        assertMatchesExpected(7, "", "CC", 6, 7, "", "CC"); // 6:C>CCC
+        assertMatchesExpected(7, "", "CCC", 6, 8, "", "CCC");   // 6:C>CCCC
     }
 
     @Test
     public void ambiguousDeletions() throws Exception {
-        assertMatchesExpected(3, "CG", "", 2, 3, "GC", "");
-        assertMatchesExpected(5, "CG", "", 4, 5, "GC", "");
-        assertMatchesExpected(6, "C", "", 5, 5, "C", "");
+        assertMatchesExpected(3, "CG", "", 2, 3, "GC", ""); // 2:GCG>G
+        assertMatchesExpected(5, "CG", "", 4, 5, "GC", ""); // 4:GCG>G
+        assertMatchesExpected(6, "C", "", 5, 5, "C", "");   // 5:CC>C
     }
 
     private void assertMatchesExpected(int position, String reference, String alternate, int expectedStart,
                                        int expectedEnd, String expectedReference,
                                        String expectedAlternate) throws Exception {
         int endPosition = computeEnd(position, reference, alternate);
         Variant variant = new Variant("22", position, endPosition, reference, alternate);
-        Variant renormalized = renormalizer.process(variant);
+        IVariant renormalized = renormalizer.process(variant);
         assertNotNull(renormalized);
         assertEquals(expectedStart, renormalized.getStart());
         assertEquals(expectedEnd, renormalized.getEnd());
         assertEquals(expectedReference, renormalized.getReference());
         assertEquals(expectedAlternate, renormalized.getAlternate());
     }
-}
\ No newline at end of file
+
+    @Test
+    public void variantIsCopiedCompletely() throws Exception {
+        //given
+        int start = 3;
+        String reference = "";
+        String alternate = "CG";
+        String renormalizedAlternate = "GC";
+        int endPosition = computeEnd(start, reference, alternate);
+        Variant variant = new Variant("22", start, endPosition, reference, alternate);
+        String mainId = "rsId";
+        HashSet<String> dbsnpIds = new HashSet<>(Arrays.asList(mainId, "ssId_1", "ssId_2"));
+        variant.setDbsnpIds(dbsnpIds);
+        variant.setMainId(mainId);
+        Map<String, VariantStatistics> cohortStats = new HashMap<>();
+        cohortStats.put("ALL", new VariantStatistics(reference, alternate, VariantType.INDEL, (float) 0.2, (float) 0.3,
```

Comment:
Depending on what we decide to do in the copy/renormalization code, this call could need to change.